tool_path_planner/src/halfedge_boundary_finder.cpp

```diff
@@ -150,7 +150,7 @@ bool applyMinPointDistance(const pcl::PointCloud<pcl::PointNormal>& in, pcl::Poi
  * @param dist  The required point spacing distance
  * @return True on success, False otherwise
  */
-bool applyEqualDistance_original(const pcl::PointCloud<pcl::PointNormal>& in, pcl::PointCloud<pcl::PointNormal>& out,double dist)
+bool applyEqualDistance(const pcl::PointCloud<pcl::PointNormal>& in, pcl::PointCloud<pcl::PointNormal>& out,double dist)
```

Comment:
This subroutine needs significantly more comments describing conceptually what it is trying to do. In a few months, neither one of us will remember. If we write something now, it helps our future selves.